modules/client/src/main/java/org/apache/ignite/internal/jdbc/ConnectionProperties.java

```diff
@@ -0,0 +1,445 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.jdbc;
+
+import java.sql.SQLException;
+import org.apache.ignite.internal.client.HostAndPortRange;
+import org.apache.ignite.internal.client.ProtocolBitmaskFeature;
+
+//TODO IGNITE-15428 Refactor and implement SSL connection parameters.
+/**
+ * Provide access and manipulations with connection JDBC properties.
+ */
+public interface ConnectionProperties {
```

Comment:
Lest remove all unused properties. Looks like only schema is used now.

Reply:
I left only schema, url, query and connection timeouts (as they are present in jdbc api). I deleted the rest.